client/js/TemplateContext.js

```diff
@@ -97,7 +100,7 @@ module.exports = {
     
     IconDataJs( p ) { return p.name ? `data-js="${p.name}"` : `` },
 
-    ImageSrc( name ) { return `https://storage.googleapis.com/mega-poetry-9665/${name}` },
+    ImageSrc( name ) { return `https://storage.googleapis.com/five-gallon/${name}` },
```

Comment:
We will put the bucket name in the config in the near future.  Make a note of it, or checkout the config directory, package.json in 

https://github.com/FutureDaysSoftware/MaricannPortal/tree/configRefinement